src/screens/GroupInfo/elements/Header.js

```diff
@@ -37,6 +38,6 @@ const styles = StyleSheet.create({
     fontSize: normalizeFontSize(16),
     fontFamily: fonts.inter[600],
     lineHeight: normalizeFontSize(19.36),
-    color: '#000'
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hardcoded color values with standardized ones from a central theme file improves maintainability and consistency across the application. Good job!
```diff
-    color: '#000'
+    color: COLORS.black
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->